Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1309,33 +1310,17 @@ inline bool BuilderConverter::createGridPosition(const CSSValue& value, GridPosi
         return true;
     }
 
-    auto& values = downcast<CSSValueList>(value);
-    ASSERT(values.length());
-
-    auto it = values.begin();
-    auto* currentValue = &downcast<CSSPrimitiveValue>(*it);
-    bool isSpanPosition = false;
-    if (currentValue->valueID() == CSSValueSpan) {
-        isSpanPosition = true;
-        ++it;
-        currentValue = it != values.end() ? &downcast<CSSPrimitiveValue>(*it) : nullptr;
-    }
-
-    int gridLineNumber = 0;
-    if (currentValue && currentValue->isInteger()) {
-        gridLineNumber = currentValue->intValue();
-        ++it;
-        currentValue = it != values.end() ? &downcast<CSSPrimitiveValue>(*it) : nullptr;
-    }
+    auto& gridLineValue = downcast<CSSGridLineValue>(value);
+    CSSPrimitiveValue* uncheckedSpanValue = gridLineValue.spanValue();
```

Comment:
We should not have unsafe raw pointer on the stack. They should be stored in a `RefPtr`.